dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/RenameConductRBundleMojo.scala

```diff
@@ -0,0 +1,72 @@
+package com.lightbend.lagom.maven
+
+import java.io.File
+import java.util.Locale
+import java.util.zip.ZipFile
+import javax.inject.Inject
+
+import org.apache.maven.plugin.AbstractMojo
+import better.files.{ File => ScalaFile, _ }
+
+import scala.beans.BeanProperty
+import scala.collection.JavaConverters._
+
+/**
+ * Converts a zip artifact to one named according to ConductR's conventions.
+ */
+class RenameConductRBundleMojo @Inject() (logger: MavenLoggerProxy) extends AbstractMojo {
+
+  @BeanProperty
+  var sourceConductRBundle: File = _
+
+  @BeanProperty
+  var bundleName: String = _
+
+  @BeanProperty
+  var outputDirectory: File = _
+
+  override def execute(): Unit = {
+    if (!sourceConductRBundle.exists()) {
+      logger.error(s"The source ConductR bundle, ${sourceConductRBundle.getAbsolutePath}, does not exist. Make sure you have " +
+        s"configured your build to produce a ConductR zip artifact before running lagom:renameConductRBundle, and " +
+        s"that either that artifact is located at ${sourceConductRBundle.getAbsolutePath}, or you have configured " +
+        s"sourceConductRBundle to point to it. If you are using the maven assembly plugin to produce the artifact, " +
+        s"ensure it is declared in your POM before the lagom plugin, as Maven executions for the same phase are " +
+        s"executed in the order that they appear in the POM.")
```

Comment:
There are a few unneeded usages of string interopolation. Pointing it out in case you want to remove them.